tests/test_client_methods.py

```
@@ -652,3 +652,93 @@ def test_list_tags_permission_denied(self, mock_zammad_api: Mock) -> None:
+
+    @pytest.fixture
+    def mock_zammad_api(self) -> Generator[Mock, None, None]:
+        """Mock the underlying zammad_py.ZammadAPI."""
+        with patch("mcp_zammad.client.ZammadAPI") as mock_api:
+            yield mock_api
+
+    def _make_client(self, mock_zammad_api: Mock) -> tuple[ZammadClient, Mock]:
+        """Build a client with a mocked API and return (client, mock_instance)."""
+        mock_instance = Mock()
+        mock_zammad_api.return_value = mock_instance
+        client = ZammadClient(url="https://test.zammad.com/api/v1", http_token="test-token")
```

Ruff (0.16.0)

[error] Possible hardcoded password assigned to argument: "http_token"

(S106)